ste/JobPartPlanFileName.go

```diff
@@ -189,6 +189,9 @@ func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest) {
 			MetadataLength:           uint16(len(order.BlobAttributes.Metadata)),
 			BlockSize:                blockSize,
 			BlobTagsLength:           uint16(len(order.BlobAttributes.BlobTagsString)),
+			CpkInfo:                  order.CpkOptions.CpkInfo,
```

Comment:
Maj: This is a secret and should not be written to disk, much like a SAS key. Instead we should pass it on jptm/jpm and then reconstruct it from environment during resume.

Reply:
Hey @nakulkar-msft  , 
It is just a boolean variable.
If `order.CpkOptions.CpkInfo` is true, that means we'll fetch the secrets from environment variables at the time of use only. We will not save any secrets whatsoever on the disk.

Reply:
(y) I took it to cpkOptions